pandas/core/frame.py

```diff
@@ -4704,6 +4704,7 @@ def sort_values(
         inplace=False,
         kind="quicksort",
         na_position="last",
+        ignore_index=False,
```

Comment:
```suggestion
        ignore_index: bool = False,
```

Want to add annotations for any new arguments

Reply:
thanks very much for your review! @WillAyd 

this is an interesting one, i initially did add annotation for this, however, the CI failed because of it, you could see this from `b80f380`, i have to remove annotation to make the CI green. Do you have any idea why/how to fix?

Reply:
Can you post the error?

Reply:
<img width="940" alt="Screen Shot 2019-12-23 at 10 03 38 AM" src="https://user-images.githubusercontent.com/9269816/71347856-8c12eb80-256b-11ea-951d-03f507418273.png">

@WillAyd

Reply:
Right so you should add the `ignore_index` argument to NDFrame and Series as well

Reply:
thanks! apparently it's not interesting, it's just me overlooking the change needed in `NDFrame` and `Series` 😅 

fixed! @WillAyd

Reply:
I see you added this to the other functions so I think can annotate here now?